client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/get-heading.ts

```diff
@@ -29,6 +31,10 @@ export default function getHeading( props: HeadingProps ) {
 		return translate( 'Welcome to Jetpack Search!' );
 	}
 
+	if ( isBulkDomainTransfer ) {
+		return translate( 'Your domain transfer has started' );
```

Comment:
This needs a multiple transfer version like "Your domain transfers have started":
> We need to correct the copies when it’s for one or multiple. In the mocks now the multiple reads like single in the title. 

from p9Jlb4-8kT-p2#comment-8644

Reply:
This has been fixed!